napalm_ios/ios.py

```diff
@@ -158,28 +158,26 @@ def commit_config(self, filename=None):
         self._gen_rollback_cfg()
 
         # Replace operation
-        if debug:
-            base_time = datetime.now()
-            print("check1: {}".format(base_time))
         if self.config_replace:
             if filename is None:
                 filename = self.candidate_cfg
             cfg_file = self.gen_full_path(filename)
             if not self._check_file_exists(cfg_file):
                 raise ReplaceConfigException("Candidate config file does not exist")
-            if debug:
-                print("check2 (check_file_exists)")
-                print("Time delta: {}".format(datetime.now() - base_time))
-                base_time = datetime.now()
             if self.auto_rollback_on_error:
                 cmd = 'configure replace {} force revert trigger error'.format(cfg_file)
             else:
                 cmd = 'configure replace {} force'.format(cfg_file)
-            output = self.device.send_command_expect(cmd)
-            if debug:
-                print("check3 (configure replace)")
-                print("Time delta: {}".format(datetime.now() - base_time))
-                base_time = datetime.now()
+            try:
+                current_prompt = self.device.find_prompt()
```

Comment:
No, the above is all met by the current Netmiko requirement (>=0.4.3) so we should be good.